packages/client/src/app/pages/browse/browse.component.html

```diff
@@ -20,19 +20,17 @@ <h2>Latest Works</h2>
             <div class="work-container">
                 <!--Header-->
                 <div class="work-header">
-                    <ng-container [ngSwitch]="work.meta.rating">
-                        <div matTooltip="Everyone" matTooltipClass="offprint-tooltip" class="rating everyone small" *ngSwitchDefault>E</div>
-                        <div matTooltip="Teen" matTooltipClass="offprint-tooltip" class="rating teen small" *ngSwitchCase="'Teen'">T</div>
-                        <div matTooltip="Mature" matTooltipClass="offprint-tooltip" class="rating mature small" *ngSwitchCase="'Mature'">M</div>
-                        <div matTooltip="Explicit" matTooltipClass="offprint-tooltip" class="rating explicit small" *ngSwitchCase="'Explicit'">X</div>                                
-                    </ng-container>                            
-                    <h1><a [routerLink]="['/work', work._id, work.title | slugify]">{{ work.title }}</a></h1>                            
+                    <rating-icon [rating]="work.meta.rating" [size]="'small'"></rating-icon>                          
+                    <h1 [ngSwitch]="work.kind">
+                        <a *ngSwitchCase="'ProseContent'" [routerLink]="['/prose', work._id, work.title | slugify]">{{ work.title }}</a>
+                        <a *ngSwitchCase="'PoetryContent'" [routerLink]="['/poetry', work._id, work.title | slugify]">{{ work.title }}</a>
```

Comment:
This is a bit clumsy, but seemed like the best way to handle both prose and poetry.